rbac/server/api/users.py

```diff
@@ -466,6 +728,21 @@
 
 
 @USERS_BP.get("api/users/<next_id>/proposals/confirmed")
+@doc.summary("Get confirmed proposals for a user, by their next_id.")
+@doc.description("Get confirmed proposals for a user, by their next_id.")
+@doc.consumes({"head": str}, location="query")
+@doc.consumes({"start": int}, location="query")
+@doc.consumes({"limit": int}, location="query")
+@doc.produces(
+    [{"id": str, "type": str, "object": str, "target": str, "approvers": [str]}],
```

Comment:
200 response also missing some of the fields here (like link, paging, proposal details, and head):
```
{
  "link": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/confirmed?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
  "paging": {
    "last": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/confirmed?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "next": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/confirmed?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "first": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/confirmed?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100",
    "limit": 100,
    "total": 2,
    "start": 0,
    "prev": "http://localhost:8000/api/users/9ffb601a-3fcf-4614-aeff-5a6b27551025/proposals/confirmed?head=5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526&start=0&limit=100"
  },
  "data": [
    {
      "close_reason": "I am the owner of this role",
      "target": "9ffb601a-3fcf-4614-aeff-5a6b27551025",
      "id": "795436cf-3de7-412f-a7bc-a02742012692",
      "pack_id": "",
      "assigned_approver": [
        "9ffb601a-3fcf-4614-aeff-5a6b27551025"
      ],
      "approvers": [
        "9ffb601a-3fcf-4614-aeff-5a6b27551025"
      ],
      "created_date": 1565029231,
      "object": "d661ea1f-1046-411f-91e6-b658caceb39a",
      "closed_date": 1565029232,
      "status": "CONFIRMED",
      "open_reason": "",
      "metadata": {},
      "closer": "9ffb601a-3fcf-4614-aeff-5a6b27551025",
      "opener": "9ffb601a-3fcf-4614-aeff-5a6b27551025",
      "type": "ADD_ROLE_MEMBER"
    }
  ],
  "head": "5cf062df08203d80835e96c9a8172e8f6f3c3fe95f1cbb4e4fb4056ff8e5f8350a1d4ef42e6c60a43978ca6a541f5c4a49fc057c2a585b49cb4fedaf7c840526"
}
```